src/css/content.css

```diff
@@ -0,0 +1,55 @@
+.background {
+    width: 100%;
```

Comment:
As this is the background, should we use % or vw?
% is relative to parent element whilst vw is relative viewport

Reply:
vw seems more correct conceptually, but it acts the same in this situation due to body being the parent